### PR TITLE
bpo-43798: Add source location attributes to alias.

### PR DESCRIPTION
**Pull request by msuozzo**
https://bugs.python.org/issue43798

### CODE REVIEW

File: Grammar/python.gram [hunk 1]
```diff
@@ -144,20 +144,20 @@ import_from[stmt_ty]:
 import_from_targets[asdl_alias_seq*]:
     | '(' a=import_from_as_names [','] ')' { a }
     | import_from_as_names !','
-    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, CHECK(alias_ty, _PyPegen_alias_for_star(p))) }
+    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, _PyAST_alias(CHECK(PyObject *, _PyPegen_alias_star(p)), NULL, EXTRA)) }
```
Comment (lysnikolaou) [inline Grammar/python.gram]: I think I'd prefer it if we pass `EXTRA` to `_PyPegen_alias_star` and let that construct the AST node. That's what we've been doing at other places, so let's be consistent with that.
```suggestion
    | '*' { (asdl_alias_seq*)_PyPegen_singleton_seq(p, CHECK(alias_ty, _PyPegen_alias_for_star(p, EXTRA))) }
```

**Review by lysnikolaou — COMMENTED** [review]
Hey @msuozzo and thanks for the PR! :tada:  I've left a comment that needs addressing. Also, please add a NEWS blurb or, in case you don't know how, let me know so that I can add one for you. Other than that, the PR looks good, although I'd like to see some more discussion on bpo before approving this.

File: Grammar/python.gram (hunk 1 above)
Comment (pablogsal) [reply Grammar/python.gram]: Agreed 👍

**Comment by pablogsal** [comment]
This also needs a news entry. You can add one easily [here](https://blurb-it.herokuapp.com/)

File: Grammar/python.gram (hunk 1 above)
Comment (msuozzo) [reply Grammar/python.gram]: Fair enough. Done.

File: Misc/NEWS.d/next/Core and Builtins/2021-04-10-00-01-43.bpo-43798.p_nJFM.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+ast.alias nodes now include source location metadata attributes e.g. lineno, col_offset.
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Core and Builtins/2021-04-10-00-01-43.bpo-43798.p_nJFM.rst]:
```suggestion
:class:`ast.alias` nodes now include source location metadata attributes e.g. lineno, col_offset.
```

**Review by pablogsal — APPROVED** [review]

**Review by lysnikolaou — APPROVED** [review]

**Comment by lysnikolaou** [comment]
Thanks again for the PR, @msuozzo.

**Pull request merged**